datasets/us/or/med_exclusions/crawler.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict
+from lxml import etree
+import requests
+
+from zavod import Context, helpers as h
+
+RESPONSE_DATA = '<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><soap:Body><GetListItems xmlns="http://schemas.microsoft.com/sharepoint/soap/"><listName>Documents</listName><viewName>{9070D930-A5F7-4AFC-9EF4-943645B8E724}</viewName><queryOptions><QueryOptions><IncludeAttachmentUrls>TRUE</IncludeAttachmentUrls></QueryOptions></queryOptions></GetListItems></soap:Body></soap:Envelope>'
```

Comment:
Let's format this just so it's a bit more readable
```suggestion
REQUEST_DATA = ""
    <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
        <soap:Body>
            <GetListItems xmlns="http://schemas.microsoft.com/sharepoint/soap/"><listName>Documents</listName><viewName>{9070D930-A5F7-4AFC-9EF4-943645B8E724}</viewName><queryOptions><QueryOptions><IncludeAttachmentUrls>TRUE</IncludeAttachmentUrls></QueryOptions></queryOptions></GetListItems></soap:Body></soap:Envelope>'
```
etc